.ci_support/linux_64_.yaml

```diff
@@ -5,7 +5,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 curl:
-- '7'
+- '8'
```

Comment:
I notice that this PR updates `curl` but not `libcurl` below. Is that known / intentional / desirable / even working?